tutorial_src/lea/doc/leazad.tex

```diff
@@ -0,0 +1,54 @@
+\documentclass{spiral}
+  \def\title{Liście}
+  \def\id{lea}
+  \def\contest{Zadanie przykładowe}
+  \def\desc{}
+  \def\TL{1 s}
+  \def\ML{64 MB}
+
+\begin{document}
+    \makeheader
+
+    Drzewo jest to nieskierowany nieważony graf,
+    składający się z $n$ wierzchołków i $n-1$ krawędzi.
```

Comment:
Ta definicja nie jest poprawna.

Reply:
Linijkę niżej, że charakterystyką drzewa jest jego spójność/acykliczność, co dopełnia definicję. Powinienem to zmienić?

Reply:
"Drzewem nazywamy każdy spójny las"

Reply:
To powyżej to napisałem jeszcze zanim zobaczyłem Twoją odpowiedź.
Może wystarczy "jest to SPÓJNY nieskie..."

Reply:
Mam na myśli, że ja nie zauważyłem, że intencją było, aby kolejne zdanie było dalszą częścią definicji.

Reply:
Teraz powinno być (w obu wersjach) lepiej